support/tracker_view.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+"Debugging tool for verifying the target tracking code with color masking."
+
+# from lib import color_mask, util
+import argparse
+import cv2
+from context import lib          # flake8: noqa pylint: disable=unused-import
+from lib import color_mask, util, config
+
+DEBUG = True
+
+def run(channel, config_file):
+    cfg = config.Config(filename=config_file)
+    DEBUG = cfg.get_default("debug", False)

```

Comment:
Again, I like this `DEBUG` value coming from the configuration file, however, I don't think if you set something like:

    global DEBUG

This really sets the `DEBUG` global variable, and instead just creates a local variable that overshadows it. Does this really need to be a global variable? Perhaps just setting locally is sufficient?